element_miniscope/miniscope.py

```diff
@@ -41,99 +41,242 @@ def activate(imaging_schema_name, scan_schema_name=None, *,
     global _linking_module
     _linking_module = linking_module
 
-    scan.activate(scan_schema_name, create_schema=create_schema,	
+    scan.activate(scan_schema_name, create_schema=create_schema,
```

Comment:
Is the plan still to delete the `scan` module?

Reply:
you're right. let me remove.